src/projectile.cpp

```diff
@@ -136,6 +136,10 @@ void apply_ammo_effects( const tripoint &p, const std::set<std::string> &effects
         }
     }
 
+    if( effects.count( "ACID_DROP" ) > 0 ) {
+        g->m.add_field( p, fd_acid, 1, 0 );
+    }
```

Comment:
The long term plan is to support `emit` for ammo so that mod authors can define fields in JSON